distributed/core.py

```diff
@@ -234,11 +234,20 @@ def stop():
         self.periodic_callbacks["monitor"] = pc
 
         self._last_tick = time()
-        measure_tick_interval = parse_timedelta(
+        self._tick_counter = 0
+        self._tick_count = 0
+        self._tick_count_last = time()
```

Comment:
We currently use metrics.time because of windows support.  If we want to switch to something else that's better I'm all in favor, but I hesitate to launch into this immediately.  Instead I'd rather that we kept with the standard way of doing things, and then considered switching out to monotonic en-masse.  Thoughts?